### ISSUE TITLE
Scope post-festival comms

### ISSUE THREAD

**Issue by edrushka** [MozillaFoundation/Mozfest2016_production#144]
Comms pieces that should go out after the festival:

Blogs
- Mozilla's official recap blog
- Speaker series video archive blog
- Social media recap (best of Twitter and IG)

Emails
- thank you to speakers
- thank you and recap for participants
- thank you to facilitators
- thank you to Space Wranglers
- thank you to vendors
- thank you to volunteers
- recap for Mozilla 

Surveys
- participants
- facilitators
- Space Wranglers

Long Tail (weeks to months following event)
- how to continue working/teaching/collaborating - for attendees
- send us your blogs! - for attendees
- Present on Mozilla project call 

cc @KGorr @KevZawacki  @Saallen

**Comment by SpikeUK** [comment]
(cough) thank you to volunteers (cough)
(I thank the volunteers (and vouch for every Mozillian) but it would be nice to have an "official" thank you from Mozilla.)

**Comment by edrushka** [comment]
YES! Thanks for the reminder @SpikeUK - I'll edit the first comment

**Comment by KGorr** [comment]
Reflecting on my experiences with large events like this, there is always a "ah, man, it's over! What do I do now?" feeling that sinks in after returning from such a busy/stimulating high from the event. What if we also did a couple of follow up communications (email & blog) to encourage/remind people of how they can continue working, collaborating, and teaching after MozFest? I'm thinking weeks/months afterwards.

**Comment by edrushka** [comment]
Nice idea. I'll add a "long tail" category up top for stuff like this.

**Comment by Saallen** [comment]
SRCCON ask their community to send them any blogs they publish in their post event comms. I would like us to try this too- I do gather a lot through trawling twitter but I must miss many

**Comment by edrushka** [comment]
Good call. I added to the list.

**Comment by Saallen** [comment]
@xmatthewx there should also be a follow up around Network pulse and how to stay connected.
I believe @acabunoc is planning something here

**Comment by xmatthewx** [comment]
Fab. Keep in mind that Pulse intentionally doesn't collect contact information. So, followup should be integrated with other lines of communication with attendees and facilitators. 

Attendees: 

> "Keep tabs on the great things created by our network during and after Mozfest via Network Pulse https://mzl.la/pulse"

Facilitators: TBD

**Comment by Saallen** [comment]
looping @arlissc for metrics!

**Comment by edrushka** [comment]
cc @StevieBenton

**Comment by StevieBenton** [comment]
I've been thinking about this a little bit because one of the challenges is to keep updated with all the good things that begin at the festival. 

As part of the comms - whether a stand alone item or embedded in an item that is already planned - could be some form of call to action for people who attended or who facilitated a session to some kind of sign up. They would agree for the team to check in with them, perhaps every three months for an update on their projects and ideas. These would make really good blogs and legacy content for the festival website, and potentially support Mozilla's wider comms work.

**Comment by edrushka** [comment]
I agree - a regularly scheduled email checkin (every 3 months sounds like a good interval) would be useful. It could be a simple email saying "update us on your project". @Saallen would we need a sign-up, or does our privacy policy allow us to email sessions facilitators throughout the year?

**Comment by KGorr** [comment]
Thank you email drafts [here](https://docs.google.com/document/d/1EAPIYlrNmkk0SeIbyjcknwuY-ET6L0RKpZjFELAm5R4/edit#heading=h.qmpbugxu8hyw). Need some help with a few areas - noted in the doc. Also, @edrushka, is the "recap for Mozilla" under emails meant to be a staff email? Is there a desired time frame for this to be sent?

**Comment by edrushka** [comment]
Yes, it would be good to send an email to all@mozilla, or at the very least all@mofo recapping the event and thanking people for their participation. Staff won't get the attendee recap. Desired send date would be Tues or Wed after the festival (flexible, but within a week is preferred).

**Comment by KGorr** [comment]
Thanks for clarifying @edrushka!

**Comment by StevieBenton** [comment]
Hey @edrushka here is the ticket for surveys we were discussing. Will ping you a link in Mattermost as well. 

Some thoughts that Erika and I just discussed:
- Main survey needs to be in sections and include a "skip section" button so respondents don't have to go through irrelevant content
- Survey for facilitators should include questions exploring whether they felt supported enough, whether they had everything they needed, how the number and type of attendees met their expectations, whether they achieved their aims etc
- Survey for wranglers is not a priority at this point
- A survey of the Speaker Series folks would be worthwhile, could be short as well but with open fields

**Comment by StevieBenton** [comment]
Looks as though the link to the survey isn't in this ticket, sorry about that. [It's here](https://docs.google.com/document/d/1EAPIYlrNmkk0SeIbyjcknwuY-ET6L0RKpZjFELAm5R4/edit#).

**Comment by edrushka** [comment]
@KGorr @StevieBenton I've reviewed the post-festival survey to all attendees. Do you want to take another pass and create a clean version for others to review?

**Comment by StevieBenton** [comment]
Sure. @KGorr would you like me to do this or would you like to have a go?

**Comment by KGorr** [comment]
@StevieBenton - if you can fit it in, that would be awesome! I won't be able to get to it until late today my time.

**Comment by arlissc** [comment]
@edrushka - would it be possible for me to see the post-festival survey draft as well ? Thanks so much.

**Comment by edrushka** [comment]
@arlissc the link is [here](https://docs.google.com/document/d/1Cbfw8EddiXO3_wI9DphzLa_RWb14Ps5PDOtUnP13YTE/edit), although I suggest you wait until Stevie or Kristina posts a new draft. If you have any suggestions, please coordinate with @KevZawacki. Thanks.

**Comment by StevieBenton** [comment]
Hello @edrushka @KGorr - I've gone through the survey now and resolved the comments and suggestions. It's much tighter now, so thank you for all help.

@arlissc I think it should be OK for you to take a look now.

**Comment by edrushka** [comment]
@KevZawacki please review entire survey when you have a sec. TY

**Comment by edrushka** [comment]
Colin Carruthers has agreed to test the survey as an actual first time festival attendee on Monday, and let us know if any of the questions are weird/annoying and if there was anything he wanted to give feeback on but there wasn't a space. He's not on GH yet, so @KGorr can you pls email him the link to the actual survey when it's ready?

**Comment by KevZawacki** [comment]
@edrushka Reviewed! Added a few tweaks.

**Comment by KGorr** [comment]
Flagging that there are 2 thank yous that haven't been drafted yet - Space Wranglers & vendors. I can draft both, but will need direction from the team as I'm not familiar with the roles/responsibilities involved. Thanks!

**Comment by StevieBenton** [comment]
Just published the recap blog on Medium. [It's here](https://medium.com/mozilla-festival/mozfest-2016-is-a-wrap-f19f1f877a5#.a9t88zjoz), hope you like it!
